### PR TITLE
Add documentation for MethodBodyBlock

### PR DESCRIPTION
**Pull request by MSDN-WhiteKnight**

### CODE REVIEW

File: xml/System.Reflection.Metadata/MethodBodyBlock.xml [hunk 1]
```diff
@@ -51,9 +64,9 @@
         <Parameter Name="reader" Type="System.Reflection.Metadata.BlobReader" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="reader">The blob reader to read method body.</param>
+        <summary>Creates a new instance of <c>MethodBodyBlock</c> class using the specified blob reader.</summary>
```
Comment (gewarren) [inline xml/System.Reflection.Metadata/MethodBodyBlock.xml:68]:
```suggestion
        <summary>Creates a new instance of the <see cref="T:System.Reflection.Metadata.MethodBodyBlock" /> class using the specified blob reader.</summary>
```

File: xml/System.Reflection.Metadata/MethodBodyBlock.xml [hunk 2]
```diff
@@ -51,9 +64,9 @@
...
+        <returns>A new instance of <c>MethodBodyBlock</c> class.</returns>
```
Comment (gewarren) [inline xml/System.Reflection.Metadata/MethodBodyBlock.xml:69]:
```suggestion
        <returns>A new instance of the <see cref="T:System.Reflection.Metadata.MethodBodyBlock" /> class.</returns>
```

File: xml/System.Reflection.Metadata/MethodBodyBlock.xml [hunk 3]
```diff
@@ -51,9 +64,9 @@
         <Parameter Name="reader" Type="System.Reflection.Metadata.BlobReader" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="reader">The blob reader to read method body.</param>
```
Comment (gewarren) [inline xml/System.Reflection.Metadata/MethodBodyBlock.xml:67]:
```suggestion
        <param name="reader">The blob reader to read the method body.</param>
```

File: xml/System.Reflection.Metadata/MethodBodyBlock.xml [hunk 4]
```diff
@@ -51,9 +64,9 @@
         <Parameter Name="reader" Type="System.Reflection.Metadata.BlobReader" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="reader">The blob reader to read the method body.</param>
+        <summary>Creates a new instance of the <see cref="T:System.Reflection.Metadata.MethodBodyBlock" /> class using the specified blob reader.</summary>
+        <returns>A new instance of the <see cref="T:System.Reflection.Metadata.MethodBodyBlock" /> class.</returns>
         <remarks>To be added.</remarks>
```
Comment (buyaa-n) [inline xml/System.Reflection.Metadata/MethodBodyBlock.xml:70]: NIT: normally we add the Exceptions that could be thrown, seems the method could throw  `BadImageFormatException`

**Review by buyaa-n — APPROVED** [review]
Thanks, @MSDN-WhiteKnight, left an NIT comment, overall LGTM, requested @tmat's review as an expert in this area

**Pull request merged**